src/app/ApplicationTemplate.Shared.Views/Configuration/ViewServicesConfiguration.cs

```diff
@@ -64,4 +68,22 @@ private static IServiceCollection AddMessageDialog(this IServiceCollection servi
 //+:cnd:noEmit
 		);
 	}
+
+	private static IServiceCollection AddReviewService(this IServiceCollection services)
+	{
+		return services.AddSingleton((Func<IServiceProvider, IReviewService<ReviewSettingsCustom>>)(s =>
```

Comment:
@Lee31416 Like in `PersistenceConfiguration` I think.

Reply:
the cast was useless but I forgot to register the review service, somehow the cast acted as a registration because without it the app would crash but I just added it to the <> part of the AddSingleton method